src/@[scope]/[name].md.js

```diff
@@ -0,0 +1,276 @@
+    </div>
+  </a>
+</div>
+
+<div class="grid grid-cols-1">
+  <div class="card">
+    <h2>Daily downloads</h2>
+    <h3>28d <b style="color: var(--theme-foreground);">—</b> and 7d <b style="color: var(--theme-foreground-focus);">—</b> average</h3>
+    <div style="min-height: 390px;">$\{Plot.plot({
+      width,
+      height: 400,
+      marginLeft: 0,
+      marginRight: 60,
+      x: {domain},
+      y: {label: "downloads", domain: [0, d3.quantile(downloads, 0.995, (d) => d.value)]},
+      marks: [
+        Plot.axisY({anchor: "right", label: null}),
+        Plot.areaY(downloads, {x: "date", y: "value", fillOpacity: 0.2, curve: "step"}),
+        Plot.ruleY([0]),
+        Plot.lineY(downloads, Plot.windowY({k: 7, anchor: "start", strict: true}, {x: "date", y: "value", strokeWidth: 1, stroke: "var(--theme-foreground-focus)", curve: "step"})),
+        Plot.lineY(downloads, Plot.windowY({k: 28, anchor: "start", strict: true}, {x: "date", y: "value", stroke: "var(--theme-foreground)", curve: "step"})),
+        Plot.textX(versions.filter((d) => d.date >= start), {x: "date", text: "version", href: (d) => \`https://github.com/${githubRepo}/releases/tag/v$\{d.version\}\`, target: "_blank", rotate: -90, frameAnchor: "top-right", lineAnchor: "bottom", dx: -4}),
+        Plot.ruleX(versions.filter((d) => d.date >= start), {x: "date", strokeOpacity: 0.2}),
+        Plot.tip(downloads, Plot.pointerX({x: "date", y: "value"}))
+      ]
+    })}</div>
+  </div>
+</div>
+
+<div class="grid grid-cols-1">
+  <div class="card">
+    <h2>Downloads by version</h2>
+    <h3>Last seven days; top 10 versions</h3>
+    <div>$\{Plot.plot({
+      width,
+      label: null,
+      marginLeft: 40,
+      marginRight: 60,
+      x: {axis: "top", grid: true},
+      marks: [
+        Plot.barX(versions.filter((d) => d.downloads > 0), {
+          y: "version",
+          x: "downloads",
+          sort: {y: "x", reverse: true, limit: 10}
+        }),
+        Plot.text(versions.filter((d) => d.downloads > 0), {
+          y: "version",
+          x: "downloads",
+          dx: -4,
+          text: "downloads",
+          frameAnchor: "right",
+          fill: "var(--theme-background)"
+        }),
+        Plot.ruleX([0])
+      ]
+    })}</div>
+  </div>
+</div>
+
+---
+
+<div class="grid grid-cols-4">
+  <a href=https://github.com/${githubRepo}/commits class="card">
+    <h2>Days since last commit</h2>
+    <div class="big">${utcDay.count(new Date(commits[0].date), today).toLocaleString("en-US")}</div>
+  </a>
+  <a href=https://github.com/${githubRepo}/issues class="card">
+    <h2>GitHub open issues</h2>
+    <div style="display: flex; flex-wrap: wrap; gap: 0.5rem; align-items: baseline;">
+      <div class="big">${issues.filter((d) => d.state === "open").length.toLocaleString("en-US")}</div>
+    </div>
+  </a>
+</div>
+
+~~~js
```

Comment:
maybe we don't show this if there were no commits over the last 365 days

Reply:
I think it’s still useful to show as no activity. (Sadly…)